test/wastfmt/basic.txt

```diff
@@ -0,0 +1,27 @@
+;;; TOOL: wastfmt
+(module
+  (import "foo" "bar" (func (result i32)))
```

Comment:
Yeah, I just wanted to add a basic test here to make sure things are working, adding more would be good though.